Source/WebCore/platform/LocalizedStrings.cpp

```diff
@@ -99,12 +99,6 @@ static CFBundleRef webCoreBundle()
 
 RetainPtr<CFStringRef> copyLocalizedString(CFStringRef key)
 {
-#if !PLATFORM(IOS_FAMILY)
-    // Can be called on a dispatch queue when initializing strings on iOS.
-    // See LoadWebLocalizedStrings and <rdar://problem/7902473>.
-    ASSERT(isMainThread());
-#endif
-
     static CFStringRef notFound = CFSTR("localized string not found");
 
     auto result = adoptCF(CFBundleCopyLocalizedString(webCoreBundle(), key, notFound, nullptr));
```

Comment:
webCoreBundle() uses NeverDestroyed, is that actually thread safe?

Reply:
Good point.  I think the answer is technically no, but I think if multiple threads tried to initialize it at the same time, any corruption would probably not cause any problems in practice.  May as well make it really thread safe, though.